pig-tests/type-domains/sample-universe.ion

```diff
@@ -139,10 +139,22 @@
         // - records sum variant
 
         // these will be removed and replaced
-        (product product_a one::int two::product_to_remove)
-        (record record_a (one int) (two product_to_remove))
+        (product product_a
+            one::int
+            // This reference to a typed removed in the permuted domain must not cause pig to crash:
```

Comment:
(non-blocking) nit: "This reference to a type removed in the..."

Similarly for L150 and L156